.github/workflows/main.yml

```diff
@@ -4,8 +4,8 @@ on: [push, pull_request]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
+    timeout-minutes: 30
```

Comment:
This is a little over 2x the average time our CI runs take. If this exceeds that in the future, we can tweak this.

Although... this doesn't account for a PR where we need to rebuild a bunch of images. Maybe setting the timeout specifically on the JS tests step would be better? I don't think we've seen other things frequently time out.

Reply:
I ended up making this change in d6f233b.